autoload/coc/powershell.vim

```diff
@@ -1,18 +1,34 @@
 let s:root           = expand('<sfile>:h:h:h')
 let s:is_win         = has('win32') || has('win64')
-let s:is_mac = !s:is_win && !has('win32unix')
+let s:is_mac         = !s:is_win && !has('win32unix')
       \ && (has('mac') || has('macunix') || has('gui_macvim') ||
       \   (!isdirectory('/proc') && executable('sw_vers')))
 let s:is_vim         = !has('nvim')
-let s:install_script = s:root.'/'.(s:is_win ? 'install.cmd' : 'install.ps1')
+let s:install_script = s:root.'/install.ps1'
 
-if(s:is_mac)
-    let s:install_script = 'pwsh '.s:install_script
-endif
+function! coc#powershell#install(...)
+    if(exists('a:1.flags'))
+        let s:flags = a:1.flags
+    else
+	let s:flags = ''
```

Comment:
```suggestion
        let s:flags = ''
```

Reply:
not sure how I missed this, but should be good now 😄